src/js/test/index.test.js

```diff
@@ -86,3 +86,117 @@ describe('currentScreenWidth', () => {
         expect(width).toBe(false);
     });
 });
+
+describe('withinBreakpoint', () => {
```

Comment:
`withinBreakpoints` invokes `getCurrentScreenWidth` to set the `bps` it's worth writing a test to make sure this gets called.